Install cmake files to the correct directory.

Avoid installing docs to the wrong directory (will be installed by the ebuild instead).

--- a/CMake/config.cmake
+++ b/CMake/config.cmake
@@ -69,7 +69,7 @@
 
   set(RTTR_INSTALL_FULL_LIBDIR "${CMAKE_INSTALL_FULL_LIBDIR}")
 
-  set(RTTR_CMAKE_CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/rttr/cmake")
+  set(RTTR_CMAKE_CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/rttr")
   set(RTTR_ADDITIONAL_FILES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/rttr")
 
 elseif(WIN32)
@@ -169,8 +169,4 @@
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/CMake/rttr-config-version.cmake"
             DESTINATION ${RTTR_CMAKE_CONFIG_INSTALL_DIR}
             COMPONENT Devel)
-
-    install(FILES "${LICENSE_FILE}" "${README_FILE}"
-            DESTINATION ${RTTR_ADDITIONAL_FILES_INSTALL_DIR}
-            PERMISSIONS OWNER_READ)
 endif()

